docs/getting-started/first-steps-with-celery.rst

```diff
@@ -447,19 +446,19 @@ the task id after all).
 3) Make sure that you don't have any old workers still running.
 
     It's easy to start multiple workers by accident, so make sure
-    that the previous worker is properly shutdown before you start a new one.
+    that the previous worker is properly shut down before you start a new one.
 
-    An old worker that aren't configured with the expected result backend
+    An old worker that isn't configured with the expected result backend
     may be running and is hijacking the tasks.
 
     The :option:`--pidfile <celery worker --pidfile>` argument can be set to
     an absolute path to make sure this doesn't happen.
 
 4) Make sure the client is configured with the right backend.
 
-    If for some reason the client is configured to use a different backend
-    than the worker, you won't be able to receive the result,
-    so make sure the backend is correct by inspecting it:
+    If, for some reason, the client is configured to use a different backend
+    than the worker, you won't be able to receive the result.
+    Make sure the backend is correct by inspecting it:
```

Comment:
Perhaps: 
- `Make sure the backend is configured correctly`
- `Make sure the backend is initializred correctly`

What do you think?